src/NATS.Client.KeyValueStore/NatsKVContext.cs

```diff
@@ -19,6 +19,8 @@ public enum NatsKVStorageType
 /// </summary>
 public class NatsKVContext : INatsKVContext
 {
+    private const string KvStreamPrefix = "KV_";
```

Comment:
Generally, wouldn't be a stickler on this but using `KvStreamNamePrefix` would separate it from the subject prefix `$KV` used below.